src/Controller/DefaultController.php

```diff
@@ -186,8 +188,27 @@ protected function isAutoFsAvailable()
      */
     protected function clearCache()
     {
-        $realCacheDir = $this->cacheDir;
-        $this->cacheClearer->clear($realCacheDir);
+        try {
+            $commandAndParams = [
+                'command' => 'cache:clear',
+                '--env' => 'prod'
+            ];
+            $application = new Application($this->kernel);
+            $application->setAutoExit(false);
+            $input = new ArrayInput($commandAndParams);
+            $output = new BufferedOutput();
+            $status = $application->run($input, $output);
+            if (0 == $status) {
+                $this->info('Command success: ' . $commandAndParams['command']);
+            } else {
+                $this->err('Command failure: ' . $commandAndParams['command']);
+            }
+            sleep(2);
```

Comment:
I think we might be able to get rid of this, and if we are not we should at least put a comment.
Read my next comment about redirect.

Reply:
Tried both options without success. I don't want to spend more time with this, sleep will have to do unless someone else finds a better solution. Thanks for your suggestions!

Reply:
At least a comment for the sleep please, otherwise the next one coming here will be tempted to delete it.